tests/sheet/sheets_plugin.test.ts

```diff
@@ -1075,4 +1075,14 @@ describe("sheets", () => {
     const zone = toZone("A1:J1");
     expect(model.getters.getUnboundedZone(sheetId, zone)).toEqual({ ...zone, right: undefined });
   });
+
+  test.each<string>(["A1:Z", "A2:Z", "B2:26", "B1:26"])(
```

Comment:
you could also add a test where the head (top-left) is unbounded
```suggestion
  test.each<string>(["A1:Z", "A2:Z", "B2:26", "B1:26", "A:A"])(
```

Reply:
done :) and fixed a typo in the commit title